v1/metrics.yaml

```diff
@@ -429,115 +439,860 @@ paths:
+              external: 100
+      x-request-handler:
+        - get_from_backend:
+            request:
+              uri: '{{options.host}}/registered-users/new/{project}/{granularity}/{start}/{end}'
+              headers:
+                x-client-ip: '{{x-client-ip}}'
+      x-monitor: false
+
+
+definitions:
+  listing:
+    description: The result format for listings
+    required:
+      - items
+    properties:
+      items:
+        type: array
+        items:
+          type: string
+
+  pageview-article:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            access:
+              type : string
+            article:
+              type: string
+            agent:
+              type: string
+            granularity:
+              type: string
+            timestamp:
+              # the hourly timestamp will be stored as YYYYMMDDHH
+              type: string
+            views:
+              type: integer
+              format: int64
+
+  pageview-project:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            access:
+              type : string
+            agent:
+              type: string
+            granularity:
+              type: string
+            timestamp:
+              # the hourly timestamp will be stored as YYYYMMDDHH
+              type: string
+            views:
+              type: integer
+              format: int64
+
+  pageview-tops:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type: string
+            access:
+              type : string
+            year:
+              type: string
+            month:
+              type: string
+            day:
+              type: string
+            articles:
+              # format for this is a json array: [{rank: 1, article: <<title>>, views: 123}, ...]
+              type: string
+
+  unique-devices:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            access-site:
+              type : string
+            granularity:
+              type: string
+            timestamp:
+              # the daily timestamp will be stored as YYYYMMDD
+              type: string
+            devices:
+              type: integer
+              format: int64
+
+  pagecounts-project:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            access-site:
+              type : string
+            granularity:
+              type: string
+            timestamp:
+              type: string
+            count:
+              type: integer
+              format: int64
+
+  new-pages:
+    properties:
```

Comment:
type: object

Reply:
Not used anywehre else in file - really needed?